dev/bots/test.dart

```diff
@@ -118,7 +118,7 @@ Future<Null> _pubRunTest(
   String workingDirectory, {
   String testPath,
 }) {
-  final List<String> args = <String>['run', 'test'];
+  final List<String> args = <String>['run', 'test', '-rexpanded'];
```

Comment:
Yes - to help see what tests were running on the bots. Here's the before: https://travis-ci.org/flutter/flutter/jobs/223314813#L5740, and the after: https://travis-ci.org/flutter/flutter/jobs/223317672#L5804.